ccd-definition/CaseEventToComplexTypes/Languages-hnl-legal-rep.json

```diff
@@ -47,54 +47,6 @@
     "FieldDisplayOrder": 3,
     "DisplayContext": "MANDATORY"
   },
-  {
```

Comment:
these fields are just moved to `ccd-definition/CaseEventToComplexTypes/LanguagesLRspec-hnl-legal-rep.json` because they the caseeventid is `DEFENDANT_RESPONSE_SPEC` or `CLAIMANT_RESPONSE_SPEC`